packages/block-library/src/navigation-link/edit.js

```diff
@@ -208,7 +208,7 @@ function NavigationLinkEdit( {
 					'has-link': !! url,
 				} ) }
 			>
-				<div className="wp-block-navigation-link__inner">
+				<div>
```

Comment:
Can we get rid of this `<div>`?

Reply:
I don't think so as it's what the LinkControl Popover uses to position itself.